workflow/r/ds-pack/v1/packages.R

```diff
@@ -0,0 +1,22 @@
+install.packages('rlang', version = '1.0.0')
```

Comment:
This will take a lot of time. We can reduce this by passing `nCpus` to `install.packages` and installing all the packages in a single function call.